frodo.js

```diff
@@ -1,76 +1,74 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?